src/test/resources/discovery/test-manifest-sample-1.ttl

```diff
@@ -1,76 +1,54 @@
-@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
-@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
-@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
-@prefix earl: <http://www.w3.org/ns/earl#> .
-@prefix dcterms: <http://purl.org/dc/terms/> .
-@prefix td: <http://www.w3.org/2006/03/test-description#> .
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix earl: <http://www.w3.org/ns/earl#>
+prefix dcterms: <http://purl.org/dc/terms/>
+prefix td: <http://www.w3.org/2006/03/test-description#>
+prefix spec: <http://www.w3.org/ns/spec#>
 
-@prefix manifest: <https://example.org/manifests/test-manifest-sample-1.ttl#> .
+prefix manifest: <#>
 
-manifest:group1
-    a td:SpecificationTestCase ;
-    dcterms:title "Group 1"@en ;
-    dcterms:description "Description 1"@en ;
-    td:specificationReference <https://example.org/specification1#spec1> ;
-    dcterms:hasPart
-        <https://example.org/dummy/group1/feature1>,
-        <https://example.org/dummy/group1/feature2>,
-        <https://example.org/dummy/group1/feature3>
-.
-
-<https://example.org/dummy/group1/feature1>
+manifest:group1-feature1
     a td:TestCase ;
-    dcterms:subject "MUST" ;
+    spec:requirementReference <https://example.org/specification1#spec1> ;
     td:reviewStatus td:accepted ;
-    dcterms:title "Group 1 Test case 1" .
+    dcterms:title "Group 1 Test case 1" ;
```

Comment:
Something to consider.. besides whether a spec requirement having a schema:name, we should perhaps use the same property?